### ISSUE TITLE
About loss func

### ISSUE THREAD

**Issue by DPCCCC** [NKUhealong/H2Former#7]
loss = ce_loss_func(out, batch_labels.long()) + dice_loss_func(out, batch_labels.long())
            #loss_margin = torch.mean(1-(outputs_soft[:,1,:,:] - outputs_soft[:,0,:,:])*batch_labels)
            loss = loss# + loss_margin   
在idrid-train.py中，您的损失函数中注释掉的这个损失函数loss_margin的作用是什么？应该如何使用呢？
